aiida_quantumespresso/workflows/q2r/base.py

```diff
@@ -21,46 +19,46 @@ class Q2rBaseWorkChain(BaseRestartWorkChain):
 
     @classmethod
     def define(cls, spec):
+        # yapf: disable
         super(Q2rBaseWorkChain, cls).define(spec)
-        spec.input('code', valid_type=orm.Code)
-        spec.input('parent_folder', valid_type=orm.FolderData)
-        spec.input('parameters', valid_type=orm.Dict, required=False)
-        spec.input('settings', valid_type=orm.Dict, required=False)
-        spec.input('options', valid_type=orm.Dict, required=False)
+        spec.expose_inputs(Q2rCalculation, namespace='q2r')
+        spec.expose_outputs(Q2rCalculation, exclude=('remote_folder',))
```

Comment:
Idem -- Do you also want to exclude the output `retrieved` (inherited from `CalcJob`)?